pkg/smb/nodeserver_test.go

```diff
@@ -58,6 +71,12 @@ func TestNodeStageVolume(t *testing.T) {
 		desc        string
 		req         csi.NodeStageVolumeRequest
 		expectedErr testutil.TestError
+
+		// use the below fields only for flaky Windows error messages
+
+		// if true, use substring to match errors
+		isWindowsErrorFlaky bool
```

Comment:
what about only use `requiredErrorSubstr`? it seems `isWindowsErrorFlaky` could be removed